samples/bluetooth/hci_usb/sample.yaml

```diff
@@ -10,7 +10,7 @@ tests:
       - usb
       - bluetooth
     # FIXME: exclude due to build error
-    platform_exclude: 96b_carbon
+    platform_exclude: 96b_carbon stm32l562e_dk
```

Comment:
Is this because of USB?

Reply:
Yes. Since the platform is running a 2-chip configuration where the controller part is running on BlueNRG-MS using an SPI interface, and the STM32L4xxxx is running the host part connected via SPI to the controller, therefore, the hci_usb protocol does not apply to this platform.